daffodil-test/src/test/resources/org/apache/daffodil/section23/dfdl_functions/home_schema.dfdl.xsd

```diff
@@ -63,4 +63,6 @@
     </xs:complexType>
   </xs:element>
 
+  <xs:element name="e5" type="xs:string" dfdl:inputValueCalc="{ fn:namespace-uri(0) }"/>
```

Comment:
I think if you pass the argType as NodeInfo.Exists instead of NodeInfo.AnyAtomic this won't even compile, which I think is the behavior we want. namespace-uri should really only work when passed a node.

Reply:
OK, I'll try that!

Reply:
The change didn't help the test case (it still passed an int), but at least the NodeInfo.Exists argType is better documentation.

Reply:
So this sure seems like it should be a type error to me. If this doesn't cause a type error, then there's a separate bug, because passing any value to something that reqiures a NodeInfo.Exists should cause a type error. You can create a separate ticket for that, and add a comment here mentioning that ticket id so that we can find this test associated with that bug.